### DISCUSSION TITLE
Provide path in table output

### DISCUSSION THREAD

**Discussion by raoganeshr** [aquasecurity/trivy#4773] [Ideas]
### Description

Currently the table output does not have the path. We use the table output since its easy for the team members to check what failed etc instead of trying to format the json file.

However, the table lacks the path where the package is found. This sometimes leads to confusion leading to teams marking the finding as false positive till we create the json output and actually check the path.

### Target

None

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
duplicate of https://github.com/aquasecurity/trivy/discussions/4249
